tools/nl/embeddings/data/preindex/bio/duplicate_names.csv

```diff
@@ -1,20 +1,82 @@
 PreferredSV,DroppedSV,DuplicateName
 <-referenceSNPClusterID{typeOf:GeneticVariantGeneAssociation}->geneSymbol,<-geneSymbol{typeOf:GeneticVariantGeneAssociation}->referenceSNPClusterID,GeneticVariantGeneAssociation
 <-diseaseOntologyID{typeOf:DiseaseGeneAssociation}->geneID,<-geneID{typeOf:DiseaseGeneAssociation}->diseaseOntologyID,DiseaseGeneAssociation
-Amount_EconomicActivity_GrossDomesticProduction_Nominal,dc/topic/GDP,GDP
-dc/topic/Mortality,dc/topic/WHOMortality,Mortality
+Amount_EconomicActivity_GrossDomesticProduction_Nominal_PerCapita,Amount_EconomicActivity_GrossDomesticProduction_Nominal_AsAFractionOf_Count_Person,nominal Gross Domestic Product (GDP) per capita
+Annual_Amount_Emissions_Acetaldehyde_SCC_3_IndustrialProcesses,Annual_Amount_Emissions_Acetaldehyde_SCC_23_IndustrialProcesses,amount of acetaldehyde emissions from Industrial Processes
+Annual_Amount_Emissions_Ammonia_SCC_3_IndustrialProcesses,Annual_Amount_Emissions_Ammonia_SCC_23_IndustrialProcesses,amount of ammonia emissions from Industrial Processes

```

Comment:
wonder why there are more duplicate names now?

Reply:
these duplicate names look the same as the duplicate names in the medium index & they are svs that are newly added to sv descriptions for bio in this PR. So my guess is that these are all SVs that did not exist before when bio index was originally created.